### ISSUE TITLE
Date Format Not Following System?

### ISSUE THREAD

**Issue by metal450** [portfolio-performance/portfolio#1697]
Running PortfolioPerformance 0.47.0 on Linux (Kubuntu 20.04), I can't seem to get the date formats to change (or, to follow the system setting). In the OS, the date format is set to "en_SE," corresponding to yyyy-mm-dd HH:mm

![2020-08-26 13 08 25](https://user-images.githubusercontent.com/1877409/91351538-50d15800-e79d-11ea-8ef3-946e8685afb4.png)

However, in PortfolioPerformance, everything is still displayed as dd mmm yyyy, HH:mm

![2020-08-26 13 08 25](https://user-images.githubusercontent.com/1877409/91351693-9130d600-e79d-11ea-8b80-032122dc5835.png)

I've tried explicitly specifying a format by launching with `./PortfolioPerformance -nl en_SE` (as suggested https://forum.portfolio-performance.info/t/date-format-in-english/9738/5), but it seems to make no difference.

How can I configure PP to display dates in ISO-compliant format: YYYY-MM-DD HH:MM (24h) ?

Thanks!

**Comment by Ragas13** [comment]
@metal450 
PP as Java application follows the Java country locals:
https://www.oracle.com/java/technologies/javase/jdk8-jre8-suported-locales.html

Java 8 does not know en_se for formatting dates/numbers.

**Comment by metal450** [comment]
Understood (as far as why en_SE doesn't work).  Still though, I'm unclear about how I might get it to display dates in ISO-compliant format?

**Comment by flywire** [comment]
Interesting. I suspect PP is the same as many apps using locale, not just in java. Still, I agree with @metal450 that specific format settings should overwrite region (locale).

**Comment by metal450** [comment]
Note that it also seems not to respect the system time/date settings on Windows, so this problem exists in both OSs:

![2023-04-09 10 54 09](https://user-images.githubusercontent.com/1877409/230788676-adc20ceb-ab40-4e4e-b654-ea2c8d7a4cfc.png)

This applies to 0.62.0, the current version. Given that it has persisted for over 2.5 years, any possibility of a solution?

**Comment by Sn1kk3r5** [comment]
It wasn't of much interest for >4 years.

In general, I would completely agree. The issue is almost no one looks in those really old request.

We do have a feature short list and paper cuts, which should help getting a better overview.

You seem to care for this issue, if you feel better, I'm happy to reopen it again.

**Comment by metal450** [comment]
Fair enough, I certainly agree that there are many much higher priorities & old stuff is unlikely to be looked at in the foreseeable future. And I'm all for cleaning house on issues that are truly resolved but were never closed, superseded by other fixes, etc. It just seemed like for items that weren't actually addressed, what's the harm of not having them be lost/discarded? Long-term wishlists are a thing.

As for this one specifically, it would be nice, but I'm not dying for it. As you say, there are other higher priorities.

**Comment by flywire** [comment]
Action seems reasonable provided it is realised `not planned` includes requested.

> It wasn't of much interest for >4 years.

You can't know that, all you can say is there have been few comments.

A lot of default Java leaves bad artefacts when it flows through to the UI.

**Comment by Sn1kk3r5** [comment]
@flywire 
Indeed and technically right, but I'm not saying this for a fact. 
But at the same time, with have not received more feedback from others, or double, tripple, quotes.

I know this is bothering to a certain extend.

**Comment by HuckleHubbing** [comment]
I'm looking into using Portfolio Performance. This was the first thing I noticed. I've set all my computer's date displays to YYYY-MM-DD. But cannot set PP's to this. When looking at transaction history, or any history, this is extremely useful to have in the ISO 8601 formatting. Most applications handle this by allowing users to explicitly state the date format, regardless of the region or language selected. 

Given that all of PP's importing and exporting defaults to ISO 8601 formatting (YYYY-MM-DD), it makes sense that there would be a display for this, as well.
